todo_app/app.py

```diff
@@ -1,4 +1,6 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect
+import todo_app.data.session_items
```

Comment:
You can write 
```suggestion
from todo_app.data import session_items
```

So that you can refer to it as just `session_items` in your code rather than the longwinded `todo_app.data.session_items`

Reply:
Done